pkg/util/kubernetes/kubelet/containers.go

```diff
@@ -185,3 +190,15 @@ func parseContainerReadiness(status ContainerStatus, pod *Pod) string {
 	}
 	return containers.ContainerUnhealthy
 }
+
+// KubeContainerIDToEntityID builds an entity ID from a container ID coming from
+// the pod status (i.e. including the <runtime>:// prefix).
+func KubeContainerIDToEntityID(ctrID string) (string, error) {
+	if strings.Contains(ctrID, "://") {
```

Comment:
You could simplify here, in the same style like you did elsewhere:
```go
if !strings.Contains(ctrID, "://") {
    return "", fmt.Errorf("can't extract an entity ID from container ID %s", ctrID)
}
// ...
```

Reply:
In fact you can even simplify further by doing directly

```go
strings.LastIndexAny(s, "://")
```

which allows to remove the `Contains` and is safer if somehow the container id contains a `/` 👍

Reply:
❤️